### ISSUE TITLE
The Swift example app logic for checking the first time that the `hmiLevel` is `FULL` is broken

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/sdl_ios#1554]
### Bug Report
The Swift example app code for finding the first time that the `hmiLevel` is `FULL` is broken as the following code never gets called in the `ProxyManager` class: 

```
if newLevel == .full && firstHMILevelState != .full {
    // This is our first time in a `FULL` state.
    firstHMILevelState = newLevel
}
```

This logic doesn't seem to be in the Obj-C example app so it either the logic needs to get fixed in both example apps or removed from the Swift example app.

##### Reproduction Steps
1. Connect Swift example app to a head unit.

##### Expected Behavior
The code above should be called the very first time the `hmiLevel` is `FULL`.

##### Observed Behavior
The code above is never called.

##### OS & Version Information
* iOS Version: n/a
* SDL iOS Version: 6.5
* Testing Against: SYNC 4

**Comment by lapinskijw** [comment]
@NicoleYarroch Unless there is a best practice we should be including here regarding first `hmiLevel` `FULL` I believe it would be best to remove the code for a couple reasons.

1. As you said, this logic is not in the Obj-C example app.
2. The only action this code accomplishes is setting the fristHMILevelState to full, which occurs in `hmiLevel:didChangeToLevel:`.

**Comment by NicoleYarroch** [comment]
@lapinskijw I agree with removing the code from the Swift app based on the situations you outlined.

**Comment by theresalech** [comment]
Closed via #1574
